src/components/Header/HeaderNavDesktop.vue

```diff
@@ -32,3 +32,37 @@ export default defineComponent({
   }
 }
 </style>
+<!-- <script>
+import { defineComponent } from "vue";
+import HeaderLogo from "@/components/header/HeaderLogo.vue";
+import HeaderNavMenu from "@/components/header/HeaderNavMenu.vue";
+import HeaderNavRedirect from "@/components/header/HeaderNavRedirect.vue";
+
```

Comment:
No need to leave commented out code - we have git history for this if it's some old version of this component 😄